drivers/sd-card/sd.h

```diff
@@ -87,7 +93,7 @@ struct sd_init_param {
  */
 struct sd_desc {
 	/** Descriptor of an initialized SPI channel */
-	struct spi_desc *spi_desc;
+	struct spi_desc	*spi_desc;
```

Comment:
Is this change wanted or just a small error?

Reply:
It was wanted. I replaced a space with a tab